x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/last_value.tsx

```diff
@@ -0,0 +1,310 @@
+    }
+    if (invalidSortFieldMessage) {
+      errorMessages = [invalidSortFieldMessage];
+    }
+    return errorMessages.length ? errorMessages : undefined;
+  },
+  buildColumn({ field, previousColumn, indexPattern }) {
+    const sortField = isTimeFieldNameDateField(indexPattern)
+      ? indexPattern.timeFieldName
+      : indexPattern.fields.find((f) => f.type === 'date')?.name;
+
+    if (!sortField) {
+      throw new Error(
+        i18n.translate('xpack.lens.functions.lastValue.missingSortField', {
+          defaultMessage: 'This index pattern does not contain any date fields',
+        })
+      );
+    }
+
+    const sortOrder =
+      (previousColumn?.params &&
+        'sortOrder' in previousColumn.params &&
+        previousColumn?.params?.sortOrder) ||
+      'desc';
+
+    return {
+      label: ofName(field.displayName),
+      dataType: field.type as DataType,
+      operationType: 'last_value',
+      isBucketed: false,
+      scale: field.type === 'string' ? 'ordinal' : 'ratio',
+      sourceField: field.name,
+      params: {
+        sortOrder,
+        sortField,
+      },
+    };
+  },
+  toEsAggsConfig: (column, columnId) => ({
+    id: columnId,
+    enabled: true,
+    schema: 'metric',
+    type: 'top_hits',
+    params: {
+      field: column.sourceField,
+      aggregate: 'concat',
+      size: 1,
+      sortOrder: column.params.sortOrder,
+      sortField: column.params.sortField,
+    },
+  }),
+
+  isTransferable: (column, newIndexPattern) => {
+    const newField = newIndexPattern.getFieldByName(column.sourceField);
+    const newTimeField = newIndexPattern.getFieldByName(column.params.sortField);
+    return Boolean(
+      newField &&
+        newField.type === column.dataType &&
+        !newField.aggregationRestrictions &&
+        newTimeField?.type === 'date'
+    );
+  },
+
+  paramEditor: ({ state, setState, currentColumn, layerId }) => {
+    const currentIndexPattern = state.indexPatterns[state.layers[layerId].indexPatternId];
+    const dateFields = getDateFields(currentIndexPattern);
+    const sortOrderButtons = [
+      {
+        id: `lns-lastValue-ascending`,
+        label: 'Ascending',
+        value: 'asc',
+      },
+      {
+        id: `lns-lastValue-descending`,
+        label: 'Descending',
+        value: 'desc',
+      },
+    ];
+    const isSortFieldInvalid = !!getInvalidSortFieldMessage(
+      currentColumn.params.sortField,
+      currentIndexPattern
+    );
+    return (
+      <>
+        <EuiFormRow
+          label={i18n.translate('xpack.lens.indexPattern.lastValue.sortField', {
+            defaultMessage: 'Sort by date field',
+          })}
+          display="columnCompressed"
+          fullWidth
+          error={i18n.translate('xpack.lens.indexPattern.sortField.invalid', {
+            defaultMessage: 'Invalid field. Check your index pattern or pick another field.',
+          })}
+          isInvalid={isSortFieldInvalid}
+        >
+          <EuiComboBox
```

Comment:
I can't blur this combobox on click outside. Is it configured differently than usual? Something seems wrong.

Reply:
I cannot reproduce it. Can you send me a video or let's talk during sync.